source/code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -229,10 +229,9 @@ namespace PlayFab
             // Custom data received is a pointer to our api object
             PlayFabQoSApi* api = reinterpret_cast<PlayFabQoSApi*>(customData);
 
-            auto a = result.QosServers;
-            for (auto it = a.begin(); it != a.end(); ++it)
+            for(const QosServer& server :result.QosServers)
             {
-                api->regionMap[it->Region] = move(it->ServerUrl);
+                api->regionMap[server.Region] = move(server.ServerUrl);
```

Comment:
>move [](start = 48, length = 4)

I don't think you can (or want to) move this out of the server variable since it's a const reference.

You'll have to do a copy here but it'll still be N less copies than when you were copying the whole list before .